andela/rivendell-ah-client
# #160283233 set up axios instance

#### What does this PR do?
- Add axios for making API requests

#### Description of Task to be completed?
- Install axios as project dependencies
- Set up and axios instance
- Add a function for making API requests using axios

#### How should this be manually tested?
- Clone the repository and in your terminal `npm install` to install dependencies, then run `npm run dev` to start the application in development mode
- When the browser open try clicking the button `Secondary` and check the console for messages

#### Any background context you want to provide?
- Two functions are defined one is for starting an axios instances which can be use out of the box
- The function apiCalls is accepts the axios instance, and other parameters (header method, body) and make an api call using axios

#### What are the relevant pivotal tracker stories?
#160283233

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/30448913/45148086-628b5080-b1be-11e8-9a35-aaf9745b4926.png)
![image](https://user-images.githubusercontent.com/30448913/45153947-bea9a100-b1cd-11e8-9d23-3bde07bd70e4.png)


#### Questions:
